src/main/scala/com/mozilla/spark/sql/hyperloglog/aggregates/HyperLogLogMerge.scala

```diff
@@ -80,3 +80,20 @@ class HyperLogLogMerge extends UserDefinedAggregateFunction {
     com.twitter.algebird.HyperLogLog.toBytes(state)
   }
 }
+
+/**
+ * This class adds the capability to take in
```

Comment:
It might be worth mentioning what the use case is within the comment as it's not entirely clear otherwise.